aeat_mapping.py

```diff
@@ -31,8 +32,20 @@ class BaseInvoiceMapper(Model):
     issue_date = attrgetter('invoice_date')
     invoice_kind = attrgetter('sii_operation_key')
     rectified_invoice_kind = tools.fixed_value('I')
-    not_exempt_kind = attrgetter('sii_subjected_key')
-    exempt_kind = attrgetter('sii_excemption_key')
+
+    def not_exempt_kind(self, tax):
+        return attrgetter('sii_subjected_key')(tax)
+
+    def exempt_kind(self, tax):
+        return attrgetter('sii_exemption_cause')(tax)
+
+    def not_subject(self, invoice):
+        base = 0
+        for line in invoice.lines:
+            for tax in line.taxes:
+                if tax.sii_exemption_cause == 'NotSubject':
+                    base += attrgetter('amount')(line)
```

Comment:
It seems to me that there should be a "break" after "base += ....". After all, we don't want to sum the amount twice if there are two taxes with "NotSubject".

Though I understand this should never happen...

Reply:
> It seems to me that there should be a "break" after "base += ....". After all, we don't want to sum the amount twice if there are two taxes with "NotSubject".
> 
> Though I understand this should never happen...

It should, but if there are more than one tax defined as "NotSubject" we need to sum together.

The wrong is that the function is called to much times. I move out of the taxes for.